assets/dev/js/editor/elements/views/a-container.js

```diff
@@ -0,0 +1,120 @@
+import AEmptyView from 'elementor-elements/views/container/a-empty-view';
```

Comment:
Move in the `atomic-widgets` folder.

Reply:
This file needs to be A LOT bigger. Check sections/containers to see all the methods that need to be implemented.